src/www/app/outline_server_repository/index.ts

```diff
@@ -162,9 +179,19 @@ export class OutlineServerRepository implements ServerRepository {
     }
   }
 
+  // NOTE: For extracting a name that the user has explicitly set, only.
+  // (Currenly done by setting the hash on the URL/URI)
+  private serverNameFromAccessKey(accessKey: string): string {
+    return decodeURIComponent(new URL(accessKey).hash.slice(1));
```

Comment:
This is how we extract the name from ss:// keys, but we have not decided that for ssconf:// keys. In fact, there may be other things in the hash, for example: https://github.com/shadowsocks/shadowsocks-org/issues/89#issuecomment-341617034

For now, let's only extract names from static keys. Though we will need to add the name support to dynamic keys, as this is a feature widely used.

I'm inclined to have something like `#name={NAME}`, but this needs more discussion.

Reply:
Alright, sure